packages/sku-codemod/src/program/index.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'node:fs';
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { Command } from 'commander';
+import { runTransform } from '../transform/runner.js';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const { name, description, version } = JSON.parse(
+  fs.readFileSync(resolve(__dirname, '../../package.json'), 'utf-8'),
+);
```

Comment:
Adding a package.json export makes importing this info much nicer IMO, and is just generally good practice for packages. E.g.

```jsonc
// package.json
  "exports": {
    "./package.json": "./package.json"
  }
```

```suggestion
import packageJson from 'sku-codemode/package.json' with { type: 'json' };

const { name, description, version } = packageJson;
```